db/migrate/20230719154516_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.integer :comments_counter
+      t.integer :likes_counter
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] @Lul-Abdifan Great work on your migration file, however, the user should be referenced using the `author_id` instead of `user`, this is part of the project requirement, I would recommend you do the same in comments and likes tables.
![image](https://github.com/Lul-Abdifan/Blog-App/assets/61408860/38b40da6-2a19-4475-ab82-042bfd96edaf)

Reply:
@danmainah  I appreciate your constructive feedback. But what is the functionality behind changing user to the author for likes and comments?